server/backend/testspg/jwt_utils.py

```diff
@@ -12,12 +12,18 @@
 COGNITO_USERNAME = f"test-idir_{IDP_USER_GUID}@idir"
 IDIR_USERNAME = "COGUSTAF"
 
-# for testing purpose,
-# we added this user as FOM DEV delegated admin in local sql,
-# to manage role FOM REVIEWER and FOM SUBMITTER with forest client number 00001018
-COGNITO_USERNAME_BCEID = (
+# Users setup (key identifier: cognito_user_id) for testing purpose.
+# We added these user "TEST-3-LOAD-CHILD-1"(BCEID) and "PTOLLEST"(IDIR) as
+#   FOM DEV delegated admin in local flyway sql, to manage role for
+#   FOM REVIEWER and FOM SUBMITTER with forest client number 00001018
+COGNITO_USERNAME_BCEID_DELEGATED_ADMIN = (
+    # "TEST-3-LOAD-CHILD-1"(BCEID)
     "test-bceidbusiness_bda2a1e212244dc2b9f9522057mangledc58bbb@bceidbusiness"
 )
+COGNITO_USERNAME_IDIR_DELEGATED_ADMIN = (
+    # "PTOLLEST"(IDIR)
+    "test-idir_eb65e9d7828d4718aa6f4193c0mangled@idir"
```

Comment:
Hi Ian, I think this cognito_user_id needs to match what we have for Pete in the `server/flyway/local_sql/sample_V1002__add_local_user_cognito_id.sql` file, otherwise if I have a V1002 on my local, the cognito_user_id won't match.

Reply:
Hi Catherine, I think "PTOLLEST" does not need to be in V1002 script, you can remove that from your V1002 as it will be created from V1000 script.
However, I did a minor change on the value to be consistent with same_ script as previously I only follow from V1000 script from "COGUSTAF". Please update again. Thanks.
But it does look strange from original sample_ script some of them having "mangled" at the end and some of them having "mangled" in the middle and the length are different.